api/src/main/java/org/pmiops/workbench/actionaudit/targetproperties/ProfileTargetProperty.kt

```diff
@@ -17,8 +17,7 @@ constructor(override val propertyName: String, override val extractor: (Profile)
     DISABLED("disabled", { it.disabled?.toString() }),
     ABOUT_YOU("about_you", Profile::getAboutYou),
     AREA_OF_RESEARCH("area_of_research", Profile::getAreaOfResearch),
-    INSTITUTIONAL_AFFILIATIONS("institutional_affiliations",
-            { it.institutionalAffiliations?.joinToString(", ") }),
+    AFFILIATION("affiliation", { it.verifiedInstitutionalAffiliation?.toString() }),
```

Comment:
@jaycarlton just to confirm: we're free to alter these, right?  nothing depends on these names?

Reply:
Technically you can change the enum constant name, but NOT the first argument. That's what goes in the property name column in the log, so it's part of the contract.

That said, if it's truly a new field, and you don't want to mix the new values with the old ones in a query, we should add a new one and I guess have the old one always return null so it's never changed. I haven't really had to deprecate a property before, so it's worth taking a minute to discuss. Wastes a tiny bit of log space, but it won't knock over BQ.

My preference here is to leave the names as-is and in post-processing, take the first one in the list.

Reply:
ok, thanks.

adding a new field makes sense here since the format of the affiliations have changed - it would be confusing to mix them.

Reply:
👍